WootricSDK/WootricSDK/WTREvent.m

```diff
@@ -78,8 +78,12 @@ - (void)start {
 - (void)checkEligibility:(WTRApiClient *)apiClient {
   [apiClient checkEligibility:^(BOOL eligible){
     if (eligible) {
-      [apiClient authenticate:^{
-        self.completion(apiClient.settings);
+      [apiClient authenticate:^(BOOL succeded){
+        if (succeded) {
```

Comment:
Why not use a ternary? 

```
succeeded ? self.completion(apiClient.settings) : self.completion(nil);

Reply:
```
self.completion(succeeded? apiClient.settings : nil)
```
@marinhero you could even drop the extra call the self.completion.

Reply:
Oh nice, yea that's even better Ray!